app/src/main/java/com/tari/android/wallet/ui/fragment/onboarding/CreateWalletFragment.kt

```diff
@@ -141,55 +145,15 @@ class CreateWalletActivity : BaseActivity() {
         createWalletButton.visibility = View.GONE
         progressBar.visibility = View.VISIBLE
         uiHandler.postDelayed({
-            startTariViewAnimation()
-        }, CreateWallet.tariTextAnimViewDurationMs)
-    }
-
-    private fun startTariViewAnimation() {
-        val metrics = resources.displayMetrics
-        val offset = (metrics.heightPixels / 2 - tariAnimationView.height).toFloat()
-        val tariViewTranslateAnim =
-            ObjectAnimator.ofFloat(tariAnimationView, View.TRANSLATION_Y, offset)
-        tariViewTranslateAnim.interpolator = EasingInterpolator(Ease.CIRC_IN_OUT)
-
-        tariViewTranslateAnim.addListener(object : AnimatorListenerAdapter() {
-            override fun onAnimationEnd(animation: Animator?) {
-                tariAnimationView.playAnimation()
-            }
-
-            override fun onAnimationStart(animation: Animator?) {
-                tariAnimationView.animate()
-                    .scaleX(1f).scaleY(1f).setDuration(CreateWallet.tariTextAnimViewScaleDurationMs)
-                    .start()
-            }
-        })
-
-        val fadeOutViewAnim = ValueAnimator.ofFloat(1f, 0f)
-        fadeOutViewAnim.duration = Constants.UI.shortAnimDurationMs
-
-        fadeOutViewAnim.addUpdateListener { valueAnimator: ValueAnimator ->
-            val alpha = valueAnimator.animatedValue as Float
-            viewContainer.alpha = alpha
-            videoContainer.alpha = alpha
-        }
-
-        fadeOutViewAnim.addListener(object : AnimatorListenerAdapter() {
-            override fun onAnimationEnd(animation: Animator?) {
-                viewContainer.visibility = View.INVISIBLE
-            }
-        })
-
-        val animSet = AnimatorSet()
-        animSet.playTogether(tariViewTranslateAnim, fadeOutViewAnim)
-        animSet.duration = CreateWallet.tariTextAnimViewDurationMs
-        animSet.start()
+            showCreateEmojiIdView()
+        }, Constants.UI.CreateWallet.tariTextAnimViewDurationMs)
     }
 
     private fun startViewAnimation() {
```

Comment:
This function sounds like it starts a view animation, and it does, but I think it means "play the startup view animation", so something like "playStartupAnimation" or something in those lines would make more sense and make it all more readable.